src/SFA.DAS.ApplyService.Database/Script.PreDeployment1.sql

```diff
@@ -31,4 +31,36 @@ IF OBJECT_ID('dbo.QnAs', 'U') IS NOT NULL
 IF OBJECT_ID('dbo.Entities', 'U') IS NOT NULL 
   DELETE FROM dbo.Entities; 
 
-GO 
\ No newline at end of file
+GO 
+
+IF NOT EXISTS (
+  SELECT
+    *
+  FROM
+    INFORMATION_SCHEMA.COLUMNS
+  WHERE
+    TABLE_NAME = 'Apply' AND COLUMN_NAME = 'AssessorReviewStatus')
+BEGIN
+    DELETE FROM dbo.Apply
+END
+IF (EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'dbo'  AND  TABLE_NAME = 'ApplicationWorkflow'))
+    DROP TABLE ApplicationWorkflow;
+IF (EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'dbo'  AND  TABLE_NAME = 'ApplicationSequences'))
+    DROP TABLE ApplicationSequences;
+IF (EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'dbo'  AND  TABLE_NAME = 'ApplicationSections'))
+    DROP TABLE ApplicationSections;
+IF (EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'dbo' AND  TABLE_NAME = 'Applications'))
+    DROP TABLE Applications;
+IF (EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'dbo' AND  TABLE_NAME = 'WorkflowSequences'))
+    DROP TABLE WorkflowSequences;
+IF (EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'dbo' AND  TABLE_NAME = 'WorkflowSections'))
+    DROP TABLE WorkflowSections;
+IF (EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'dbo' AND  TABLE_NAME = 'Workflows'))
+    DROP TABLE Workflows;
+IF (EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'dbo' AND  TABLE_NAME = 'Assets'))
+    DROP TABLE Assets;
+IF (EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'dbo' AND  TABLE_NAME = 'QnAs'))
+    DROP TABLE QnAs;
+IF (EXISTS (SELECT * FROM INFORMATION_SCHEMA.TABLES WHERE TABLE_SCHEMA = 'dbo' AND  TABLE_NAME = 'Entities'))
+    DROP TABLE Entities;

```

Comment:
Doesn't the SQL Dacpac remove these tables if they are not within the publish script?

Reply:
This is from PR
https://github.com/SkillsFundingAgency/das-apply-service/pull/421